api/DHSC.FingertipsNext.Modules.HealthData.Repository/IHealthDataRepository.cs

```diff
@@ -9,17 +9,19 @@ Task<IEnumerable<HealthMeasureModel>> GetIndicatorDataAsync(int indicatorId,
                                                                  int[] years,
                                                                  string[] inequalities,
                                                                  DateOnly? fromDate = null,
-                                                                 DateOnly? toDate = null);
+                                                                 DateOnly? toDate = null,
+                                                                 bool includeUnpublished = false);
 
     Task<IEnumerable<DenormalisedHealthMeasureModel>> GetIndicatorDataWithQuintileBenchmarkComparisonAsync(int indicatorId,
                                                                  string[] areaCodes,
                                                                  int[] years,
                                                                  string areaTypeKey,
                                                                  string benchmarkAreaCode,
                                                                  DateOnly? fromDate = null,
-                                                                 DateOnly? toDate = null);
+                                                                 DateOnly? toDate = null,
+                                                                 bool includeUnpublished = false);
 
-    Task<IndicatorDimensionModel?> GetIndicatorDimensionAsync(int indicatorId, string[] areaCodes);
+    Task<IndicatorDimensionModel?> GetIndicatorDimensionAsync(int indicatorId, string[] areaCodes, bool includeUnpublished);
```

Comment:
SonarQube is complaining that you've not included a default value for the new parameter here (plus it's not consistent with the rest of the changes):
```suggestion
    Task<IndicatorDimensionModel?> GetIndicatorDimensionAsync(int indicatorId, string[] areaCodes, bool includeUnpublished = false);
```

Reply:
changed